### PR TITLE
[FEAT] raise ras event for node down/active

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

**Review by anpatel1 — APPROVED** [review]

**Review by yhuan47 — APPROVED** [review]

File: dai_core/src/main/java/com/intel/dai/dsapi/BootState.java [hunk 1]
```diff
@@ -9,7 +9,6 @@
  */
 public enum BootState {
     NODE_OFFLINE,
-    NODE_ONLINE,
```
Comment (paul-amonson) [inline dai_core/src/main/java/com/intel/dai/dsapi/BootState.java:12]: Why are you removing this everywhere?

**Review by paul-amonson — CHANGES_REQUESTED** [review]
Removing NODE_ONLINE should not be done. It is a valid state. i.e. Node has power but the OS is halted.

File: dai_core/src/main/java/com/intel/dai/dsapi/BootState.java (hunk 1 above)
Comment (sindhu-106) [reply dai_core/src/main/java/com/intel/dai/dsapi/BootState.java:12]: to maintain consistency Paul

**Review by paul-amonson — APPROVED** [review]

File: dai_core/src/test/java/com/intel/dai/dsimpl/voltdb/VoltDbAdapterOperationsTest.java [hunk 2]
```diff
@@ -251,7 +250,7 @@ public void informWlmAboutActiveNode() throws Exception {
                 new String[] { "127.0.0.1" },
                 new AdapterInformation("TEST", "TEST", "here", "myName", 0L), client_);
         operations.markNodeState(BootState.NODE_OFFLINE, "location", System.currentTimeMillis() * 1000, true);
-        operations.markNodeState(BootState.NODE_BOOTING, "location", System.currentTimeMillis() * 1000, true);
+        operations.markNodeState(BootState.NODE_ONLINE, "location", System.currentTimeMillis() * 1000, true);
         operations.markNodeState(BootState.NODE_ONLINE, "location", System.currentTimeMillis() * 1000, true);
```
Comment (anpatel1) [inline dai_core/src/test/java/com/intel/dai/dsimpl/voltdb/VoltDbAdapterOperationsTest.java:254]: Arent these two same lines and redundant?

**Review by yhuan47 — APPROVED** [review]

**Review by anpatel1 — APPROVED** [review]

**Pull request merged**